templates/zerver/help/alert-words.md

```diff
@@ -0,0 +1,33 @@
+# Alert Words
+Zulip allows you to define custom alert words and phrases.
+When they are used in any stream you are subscribed to, it works
+similarly to being [@-mentioned](/help/at-mention-a-team-member)
+by another user.
+
+## Adding Alert Words
+1.  Click the cog (![cog](/static/images/help/cog.png)) in the upper
+right corner of the right sidebar.
+2.  Select **Settings** from the dropdown menu that appears.
+    ![Settings dropdown](/static/images/help/settings-dropdown-menu.png)
+
+3.  Scroll down until you reach the **Custom Alert Words** section.
+
+    ![Custom Alert Words](/static/images/help/settings-custom-alerts.png)
+
+4.  Type the word/phrase you wish to be notified for into the text box,
+    then click the **Add Alert Word** button.
+
+Once these steps are completed, you should see something similar to:
```

Comment:
I think it will make it clearer if you mention that in the screenshot "Downtime" was added as an alert word.